config/crd/bases/deployment.skyscanner.net_progressiverollouts.yaml

```diff
@@ -1,10 +1,10 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Is this the official release for v1?
Should we keep it a separate commit and release/package a specific version?

Reply:
Not at all. This was auto-generated by kubebuilder - I can revert the change and try to see where/why it was generated, but judging by https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/ `v1beta1` seems to have been deprecated in v1.16?

Reply:
that confused the hell out of me :-)
my bad

Reply:
If we use `apiextensions.k8s.io/v1` it will not run in our current control cluster, so please let's skip this for a little bit.